src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, fs_name=None):
         self.cfs = cephfs.LibCephFS(rados_inst=mgr.rados)
         logger.debug("[CephFS] mounting cephfs filesystem: %s", fs_name)
         if fs_name:
-            self.cfs.mount(filesystem_name=fs_name)
+            self.cfs.mount(filesystem_name=fs_name.encode('utf-8'))
```

Comment:
The better change here would be to make the `mount` method accept strings. Change `src/pybind/cephfs/cephfs.pyx`. See also the `cstr` method in that file and its use elsewhere.

Reply:
Done